maksze/telemetry-bundle/0.3/config/packages/maksze_telemetry.yaml

```diff
@@ -0,0 +1,14 @@
+# Configuration file for the MaksZeTelemetry bundle
+#
+# For more information about the bundle settings visit:
+# https://github.com/maksze/MaksZeTelemetryBundle/blob/master/README.md
+
+maksze_telemetry:
+    yandex_metrika:
+        #- {id: any_id}
+    yandex_webmaster:
+        #- {id: any_id}
+    google_search_console:
+        #- {id: any_id}
+    google_analytics:
+        #- {id: any_id}
```

Comment:
You should consider using environment variables here.

Reply:
Yes, of course